bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -512,6 +529,21 @@ pub trait RoomInfoListener: Sync + Send {
     fn call(&self, room_info: RoomInfo);
 }
 
+#[derive(uniffi::Record)]
+pub struct RoomNotableTags {
+    is_favorite: bool,
+}
+impl From<SdkNotableTags> for RoomNotableTags {
+    fn from(value: SdkNotableTags) -> Self {
+        RoomNotableTags { is_favorite: value.is_favorite }
+    }
+}
```

Comment:
This seems like it could be replaced by a `#[cfg_attr(feature = "uniffi", derive(uniffi::Record)]` on the SDK type.

Reply:
Oh, I didn't know we're able to expose directly sdk type!

Reply:
Yeah it's only since recently that this is possible. We do it for two or three enum types, one [here](https://github.com/matrix-org/matrix-rust-sdk/blob/6e3892528cfc81cc7a56350835e7629e967670fa/crates/matrix-sdk-ui/src/timeline/pagination.rs#L360).

Reply:
@ganfra looks fine to me. (This is the reason why the namespacing on the Kotlin-side is different for those structs, btw :-))